teachopencadd/talktorials/T038_protein_ligand_interaction_prediction/talktorial.ipynb

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
<ul><li>Maybe change workflow to 'model' something similar; The workflow would include data prep, training and so on, I presume.</li><li>Simplify the second sentence.</li><li>"We will only use the information if an interaction exists or not," -&gt; This sounds a bit strange. Just say right away, you are transforming the task to a classification</li><li>introduction of FNN acronym is missing</li><li>link to rcsb.org for 4O75</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>